segmentation/utils/spatial_analysis_utils.py

```diff
@@ -35,47 +34,50 @@ def calc_dist_matrix(label_map, ret=True, path=None):
     # Create dictionary to store distance matrices per fov
     dist_matrices = dict(zip([str(i) for i in fovs], dist_mats_list))
     # If ret is true, function will directly return the dictionary, else it will save it as a file
-    if ret:
+    if path is None:
         return dist_matrices
     else:
         np.savez(path + "dist_matrices.npz", **dist_matrices)
 
 
-def get_pos_cell_labels(analysis_type, pheno=None, fov_data=None,
-                        thresh=None, fov_channel_data=None, cell_labels=None, col=None):
-    """Based on the type of the analysis, finds positive labels that the current match phenotype or identifies cells
-    with expression values for the current maker greater than the marker threshold.
+def get_pos_cell_labels(analysis_type, pheno=None, current_fov_neighborhood_data=None,
```

Comment:
Looks like maybe you just did find/replace for current_fov_data with current_fov_neighborhood_data. 

The neighborhood analysis function doens't call get_pos_cell_labels(), so changing it here doesn't make sense.